spec/truefi2/RateModel.spec

```diff
@@ -0,0 +1,20 @@
+methods {
+  poolBasicRate(address, uint256) returns uint256 envfree
+
+  liquidRatio(uint256) returns uint256 => DISPATCHER(true)
+  getWeeklyAPY() returns uint256 => DISPATCHER(true)
+  balanceOf() returns uint256 => DISPATCHER(true)
+}
+
+rule poolBasicRateIsMonotoneWrtAmount {
+  uint256 amount1;
+  uint256 amount2;
+  address pool;
```

Comment:
style nitpick: Let's move declarations as close as possible to where they're used. So `pool` should be moved to line 16, above rate1's declaration.

It reduces the amount of working memory we need to review, and makes it clear that pool is irrelevant before the line where it's used.

Aside: I think only C still requires all variables must be declared at the start, before any of them are used.

Reply:
If you have a stack of other branches based on this one, feel free to do this change in a follow-up PR after the stack is merged. I'll approve this PR as-is.